### ISSUE TITLE
Make sure all components are fully accessible prior to launch.

### ISSUE THREAD

**Issue by adamglenn** [wvuweb/wvu-design-system#2]
Every component should be accessible, including the correct ARIA-labels, ```<main>``` or ```<section>``` tags vs. ```<div>```, etc.

**Comment by adamglenn** [comment]
I've found everything I can. We can file others as individual issues as they arise.
